WTWSMS/common/traits/02_traits.txt

```diff
@@ -1093,10 +1095,15 @@ schemer = {
 theologian = {
 	lifestyle = yes
 	
-	learning = 2
-	intrigue = 1
+	learning = 4
+	diplomacy = 2
 	
 	ruler_designer_cost = 10
+	church_opinion = 15
+
+	opposites = {
+		cynical
+	}
```

Comment:
No it doesn't. New theology more often comes from personal insights and devotion, as well as intense prayer and studies of Scripture, and years of thought.

Martin Luther- who even Catholics acknowledge was a theologian -did not nail his Ninety Five Thesis because he was cynical or because he hated the Catholic Church. He wanted to reform the Catholic Church, not start a new denomination or break from the Catholics. Ultimately, though, he was forced to do so.

Gallileo Gallilea didn't just challenge the Catholics on his scientific observations. Gallileo was a devout Catholic _monk_ who argued on Scripture as much as he did on science. He was not a cynic or anti-Catholic. He proposed an alternate theological interpretation that would account for his observations of heliocentrism. There is no doubt in my mind that he prayed hard about it.